### DISCUSSION TITLE
Cannot split sidekiq-queues

### DISCUSSION THREAD

**Discussion by Lisdexamfetimon** [mastodon/mastodon#20602] [Q&A]
Hello everyone,
I tried to split sidekiq-queues to enhance performance. Unfortunately, every process handles all job types, regardless of the setting. This also results in multiple processes running "schedulers" and creating lots of dead threads.

Is there a solution for this? I have 4.0.0.rc2...

For example:
ExecStart=/home/mastodon/.rbenv/shims/bundle exec sidekiq -d pull -c 25

leads to "Warteschlangen: default, push, mailers, pull, scheduler"

Why is that? Thanks in advance...

**Comment by Gargron** [comment]
`-d` is wrong, it's `-q`

**Reply by Lisdexamfetimon** [reply]
Oh... well...
Lets just pretend I never asked, ok? 
Thank you...:D
